Add reusable Button component and integrate it

BoltUltra opened the pull request:
Introduced a new Button component with various styles and sizes defined in CSS. Integrated this Button component across multiple frontend files, replacing existing button elements to ensure a consistent look and feel.

whateverfw commented on frontend/src/components/Button/Button.jsx:
could you please create `ui` folder and put it there?

whateverfw replied:
also, can you please create `index.js` file near `Button.jsx` and re-export this button as a named export like this `export { default as Button } from "./Button"`. And import it properly everywhere

whateverfw commented on frontend/src/pages/forms/Form.jsx:
```diff
@@ -13,6 +13,7 @@ import CongratulationsModal from 'components/congratulationsModal/Congratulation
 import StyledPopup from 'components/openpositionpopup/StyledPopup';
 import { useQuery } from '@tanstack/react-query';
 import axios from 'axios';
+import Button from '../../components/Button/Button';
```
please us absolute imports here and in other places, so after putting it into new folder it should be 
`import { Button } from "components/ui/Button"`

whateverfw commented on frontend/src/pages/spotnet/dashboard/Dashboard.jsx:
```diff
@@ -218,15 +218,15 @@ export default function Component({ walletId, telegramId }) {
+
           {closePositionError && <div>Error: {closePositionError.message}</div>}
-          <button className="telegram-button" onClick={handleOpen}>
+          <Button variant="secondary" size="lg" className="" onClick={handleOpen}>
```
remove className prop if empty here in other places (if any)

whateverfw commented on frontend/src/pages/spotnet/dashboard/Dashboard.jsx:
```diff
@@ -218,15 +218,15 @@ export default function Component({ walletId, telegramId }) {
           </div>
-
-          <button className="redeem-button" onClick={() => closePositionEvent()} disabled={isClosing}>
+          <Button variant="primary" size="lg" className="" onClick={() => closePositionEvent()} disabled={isClosing}>
```
it is broken on mobile (please take a look into mobile's Figma), you should use smaller size for mobile devices

![image](https://github.com/user-attachments/assets/194c137f-2073-4431-a8d1-92c0572a480f)

Maybe the best way for our case is to handle it in css:
so, looking into figma it seems like we can have such cases:
lg is 609px for desktop and 324px for mobile 
md is 309px for desktop and 167px for mobile

BoltUltra replied:
The issue with this part layout of the dashboard, it is not responsive on mobile screen, hence the overlapping, but i have fixed it, will be making another push

whateverfw commented on frontend/src/pages/forms/Form.jsx:
![image](https://github.com/user-attachments/assets/a9963671-7b78-48ee-a524-19f78e2ad3ef)
button on form page is broken

BoltUltra commented:
This pull request introduces a new reusable `Button` component and updates various parts of the frontend to use this new component. The changes include the creation of the `Button` component in both JSX and CSS files, and replacing existing button elements with the new `Button` component across multiple files.

### Creation of the `Button` Component:
* [`frontend/src/components/Button/Button.jsx`](diffhunk://#diff-0d2f1534db023f98522667ec97753783845475fd9c5c5489aedb64406f8352c8R1-R14): Added a new `Button` component with customizable `variant`, `size`, and `className` props.
* [`frontend/src/components/Button/button.css`](diffhunk://#diff-eb678a1aa56b5fc466f2deb47034cb0ed4dea7d9f86653a1b307f2ae1df44511R1-R105): Added CSS styles for the new `Button` component, including styles for different sizes and variants.

### Integration of the `Button` Component:
* [`frontend/src/components/LendingForm.js`](diffhunk://#diff-9b577914c7fb67126ad578ff074b41d5b1bf523ee5aada1efab6ef783f7672bbR7): Replaced existing button elements with the new `Button` component. [[1]](diffhunk://#diff-9b577914c7fb67126ad578ff074b41d5b1bf523ee5aada1efab6ef783f7672bbR7) [[2]](diffhunk://#diff-9b577914c7fb67126ad578ff074b41d5b1bf523ee5aada1efab6ef783f7672bbL169-R172)
* [`frontend/src/components/Login.js`](diffhunk://#diff-dc1208f406fe9470e282e81d01ab34cc3886dc89adf9295720ebbe83be42ce32R2-R10): Updated the login form to use the new `Button` component.
* [`frontend/src/components/TelegramNotificationModal/TelegramNotificationModal.jsx`](diffhunk://#diff-63037cb946664c07c44840068feebc6387fac3a7a2a864f6cece2825814b3767R5): Integrated the new `Button` component in the Telegram Notification Modal. [[1]](diffhunk://#diff-63037cb946664c07c44840068feebc6387fac3a7a2a864f6cece2825814b3767R5) [[2]](diffhunk://#diff-63037cb946664c07c44840068feebc6387fac3a7a2a864f6cece2825814b3767L31-R37)
* [`frontend/src/components/WalletSection.jsx`](diffhunk://#diff-e4cee6334139af38fe928f9f940b0fb15892f8569b0e5f43b256eaaddd4a5cbcR2): Replaced existing button elements with the new `Button` component in the Wallet Section. [[1]](diffhunk://#diff-e4cee6334139af38fe928f9f940b0fb15892f8569b0e5f43b256eaaddd4a5cbcR2) [[2]](diffhunk://#diff-e4cee6334139af38fe928f9f940b0fb15892f8569b0e5f43b256eaaddd4a5cbcL56-R59) [[3]](diffhunk://#diff-e4cee6334139af38fe928f9f940b0fb15892f8569b0e5f43b256eaaddd4a5cbcL85-R88)
* [`frontend/src/pages/Login.js`](diffhunk://#diff-3cd90e1d0a02ac687bd900180422988f537f0b3ef9bf3c842ba51d6ad44ccc8dR2-R10): Updated the login page to use the new `Button` component.
* [`frontend/src/pages/forms/Form.jsx`](diffhunk://#diff-dea0371758195f3e8fb3dbfd2ad57060ad64ff0bd159d5ed3a60ca759be2aa8aR16): Integrated the new `Button` component in the form submission. [[1]](diffhunk://#diff-dea0371758195f3e8fb3dbfd2ad57060ad64ff0bd159d5ed3a60ca759be2aa8aR16) [[2]](diffhunk://#diff-dea0371758195f3e8fb3dbfd2ad57060ad64ff0bd159d5ed3a60ca759be2aa8aL136-R135)
* [`frontend/src/pages/spotnet/dashboard/Dashboard.jsx`](diffhunk://#diff-84ce805e0a7145ec9fa4764637d325317091ee5712de6448a5af5fb4273dd2fbR16): Replaced existing button elements with the new `Button` component on the dashboard. [[1]](diffhunk://#diff-84ce805e0a7145ec9fa4764637d325317091ee5712de6448a5af5fb4273dd2fbR16) [[2]](diffhunk://#diff-84ce805e0a7145ec9fa4764637d325317091ee5712de6448a5af5fb4273dd2fbL221-R229)

whateverfw commented on frontend/src/pages/forms/Form.jsx:
```diff
@@ -130,7 +130,7 @@ const Form = ({ walletId, setWalletId }) => {
         </div>
         <div>
           <div className="form-button-container">
```
can you please remove this container, seems like this is broken and adds some hover

BoltUltra replied:
Oh, i found the issue, the button container is valid, i have removed the hover states, so it should work just fine now after this push

whateverfw reviewed (changes requested):
Can you please apply one small fix I've mentioned? Apart from that everything seems to be great

@BoltUltra

whateverfw reviewed (approved):
@BoltUltra Looks good for me. Thanks for great a quick job

djeck1432 commented:
@BoltUltra Please, resolve merge conflict and I will merge it

djeck1432 reviewed (approved):
Looks good to me, get back to Derisk issue please
